src/error.rs

```diff
@@ -23,7 +23,7 @@ pub enum Error {
     Io(#[source] std::io::Error),
 
     #[error("Base64 Decode error: {0}")]
-    Decode(#[source] base64::DecodeError),
+    Decode(String),
```

Comment:
With the function you can keep the `base64` error here since it's not public (an external crate has no way of accessing the value within)

Reply:
Enum variants always have the visibility of their type (`pub` in this case) no?

Reply:
https://doc.rust-lang.org/reference/visibility-and-privacy.html
> Enum variants in a pub enum are also public by default

Reply:
Oh, no, enum variants are public. I was thinking of structs somehow. Nevermind that then :)